ld-relay.go

```diff
@@ -101,7 +110,39 @@ func main() {
 		}(envName, *envConfig)
 	}
 
-	wg.Wait()
+	http.Handle("/status", http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
```

Comment:
Do we want to make the port configurable? i assume it will be listening on port 80 by default..